tessellated-geometry-metadata.json

```diff
@@ -0,0 +1,9 @@
+{
```

Comment:
Just adding the signed statement (`registry/entity/fc023c4c22dcd5ae54f7cc81a313a38b96e17954f5a5cef9bfeb460dd33ab1ee.json`) file is enough. 

One can then use the https://github.com/oasisprotocol/metadata-registry-tools package to verify and get this metadata from the signed statement.